### PR TITLE
Docstring suggestions

### PR DESCRIPTION
**Pull request by trexfeathers**
### Description

- `bridge.py`
- `cache/*`
- `pantry/*`

### CODE REVIEW

File: src/geovista/pantry/__init__.py [hunk 1]
```diff
@@ -71,7 +71,9 @@ def fetch_coastlines(resolution: str | None = None) -> pv.PolyData:
 def fetch_raster(fname: str) -> Path:
     """Download, uncompress and cache a raster file.
 
-    The asset will only be downloaded and uncompressed if it is not already available.
+    If the resource is not already available in the geovista
+    :data:`geovista.cache.CACHE`, then it will be downloaded from the
+    :data:`geovista.cache.BASE_URL`.
```
Comment (trexfeathers) [inline src/geovista/pantry/__init__.py:76]: Consistency with other entries in this file

File: src/geovista/cache/__init__.py [hunk 2]
```diff
@@ -42,7 +42,7 @@
 """Environment variable to override :mod:`pooch` cache manager path."""
 
 GEOVISTA_DATA_VERSION: str = os.environ.get("GEOVISTA_DATA_VERSION", DATA_VERSION)
-"""Environment variable to override default ``geovista-data`` version."""
+"""Environment variable to override default :attr:`DATA_VERSION`."""
```
Comment (trexfeathers) [inline src/geovista/cache/__init__.py:45]: I would prefer to combine both the old and the new docstring, but that violates line-length limits. Both `GEOVISTA_DATA_VERSION` an `DATA_VERSION` definitely need to be documented as they are important user tools, but I felt it was important to make their relationship clear - the current API documentation reads quite confusingly.

File: src/geovista/bridge.py [hunk 3]
```diff
@@ -901,11 +925,8 @@ def from_unstructured(
         unstructured mesh. This is represented in terms of indices into the
         provided `xs` and `ys` mesh geometry.
 
-        Note that, the `connectivity` must define a mesh comprised of faces
-        based on the same primitive shape e.g., a mesh of triangles, or a mesh
-        of quads etc. Support is not yet provided for mixed face meshes. Also,
```
Comment (trexfeathers) [inline src/geovista/bridge.py:906]: I believe this is no longer true? Hence I removed it.

File: src/geovista/bridge.py [hunk 4]
```diff
@@ -768,18 +781,29 @@ def from_tiff(
...
+
+        Plot the result!
+
+        >>> plotter = GeoPlotter()
+        >>> _ = plotter.add_base_layer(texture=natural_earth_1())
+        >>> _ = plotter.add_coastlines()
+        >>> _ = plotter.add_mesh(mesh, rgb=True)
+        >>> plotter.view_poi()
+        >>> plotter.camera.zoom(40)
+        >>> plotter.show()
```
Comment (trexfeathers) [inline src/geovista/bridge.py:806]: https://geovista--841.org.readthedocs.build/en/841/reference/generated/api/geovista/bridge/index.html#geovista.bridge.Transform.from_tiff

File: src/geovista/cache/__init__.py (hunk 2 above)
Comment (bjlittle) [reply src/geovista/cache/__init__.py:45]: @trexfeathers All of the public facing `goevista` environment variables e.g., `GEOVISTA_x`, will be documented in a separate reference section ... which is next on my hit list.

Hopefully that should clear up matters somewhat and we can cross-reference to those environment variable definitions from where ever they're used 👍

File: src/geovista/bridge.py (hunk 4 above)
Comment (bjlittle) [reply src/geovista/bridge.py:806]: Fun times, right ?

I'm lovin' the rich interactive docstring rendering, it's just lush 😍 

As an aside, you wouldn't fancy making the coastlines white ? i.e., `_ = plotter.add_coastlines(color="white")`

File: src/geovista/bridge.py [hunk 5]
```diff
@@ -901,11 +925,8 @@ def from_unstructured(
         unstructured mesh. This is represented in terms of indices into the
         provided `xs` and `ys` mesh geometry.
 
-        Note that, the `connectivity` must define a mesh comprised of faces
-        based on the same primitive shape e.g., a mesh of triangles, or a mesh
-        of quads etc. Support is not yet provided for mixed face meshes. Also,
-        any optional mesh `data` provided must be in the same order as the mesh
-        face `connectivity`.
+        Note that any optional mesh `data` provided must be in the same order
+        as the mesh face `connectivity`.
```
Comment (bjlittle) [inline src/geovista/bridge.py:929]: @trexfeathers ... or the order of the points, as data can be on points or cells 👍

**Review by bjlittle — CHANGES_REQUESTED** [review]
@trexfeathers Big ❤️ 

Thanks for taking the time to wade though the docstring, priceless contribution 🍻 

Just some super minor comments to service and then this is good to go 🚀

**Review by bjlittle — APPROVED** [review]

**Pull request merged**